src/cargo/ops/cargo_new.rs

```diff
@@ -809,11 +809,17 @@ fn main() {
 "
         } else {
             b"\
+pub fn add(left: usize, right: usize) -> usize {
+    left + right
+}
+
 #[cfg(test)]
 mod tests {
+    use super::*;
+    
```

Comment:
It looks like there is extra whitespace here that is causing the error. That is a super odd thing.
In `src/cargo/ops/cargo_new.rs`
![Screen Shot 2022-05-27 at 11 06 38](https://user-images.githubusercontent.com/23045215/170737364-73b99e80-3e9b-45f6-b8da-5b99968b38be.png)
in `tests/testsuite/init/auto_git/out/src/lib.rs`
![Screen Shot 2022-05-27 at 11 06 54](https://user-images.githubusercontent.com/23045215/170737361-f6a70d92-38ee-44da-a487-364eb5a38c5e.png)
From failed tests
![Screen Shot 2022-05-27 at 11 09 10](https://user-images.githubusercontent.com/23045215/170737614-1ea23d3a-dfd8-494a-a401-6d0f6d9086d8.png)

Reply:
Yep, thanks for pointing it out!